modules/ROOT/pages/lb-mapping-rules.adoc

```diff
@@ -17,6 +17,8 @@ By using proxy rules, you can map a domain or subdomain to one of your Mule appl
 A pattern is a string that defines a template for matching an input text. Whatever is placed into curly brackets (`{   }`) is treated as a variable.
 These variables can contain only letters (a-z) and cannot contain any other characters, such as digits, slashes, etc. The variable values can contain the characters`a-z0-1.&?-_`, but can't contain slashes.
```

Comment:
Only lowercase letters, or both lowercase and uppercase?

Reply:
What is the difference between a variable, which can contain only letters, and a variable value, which can contain [the string? or do you mean a-z, 0 or 1, ., $, 7, -, and _?] `a-z0-1.$7-_`?